packages/flutter_tools/test/general.shard/crash_reporting_test.dart

```diff
@@ -13,26 +13,30 @@ import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/doctor.dart';
 import 'package:flutter_tools/src/project.dart';
 import 'package:flutter_tools/src/reporting/crash_reporting.dart';
-import 'package:flutter_tools/src/reporting/reporting.dart';
 import 'package:http/http.dart';
 import 'package:http/testing.dart';
 import 'package:test/fake.dart';
+import 'package:unified_analytics/unified_analytics.dart';
 
 import '../src/common.dart';
 import '../src/fake_process_manager.dart';
+import '../src/fakes.dart';
 
 void main() {
   late BufferLogger logger;
-  late FileSystem fs;
-  late TestUsage testUsage;
+  late MemoryFileSystem fs;
   late Platform platform;
   late OperatingSystemUtils operatingSystemUtils;
   late StackTrace stackTrace;
+  late FakeAnalytics fakeAnalytics;
 
   setUp(() async {
     logger = BufferLogger.test();
     fs = MemoryFileSystem.test();
-    testUsage = TestUsage();
+    fakeAnalytics = getInitializedFakeAnalyticsInstance(
+      fs: fs,
+      fakeFlutterVersion: FakeFlutterVersion(),
+    );
```

Comment:
https://github.com/flutter/flutter/blob/c1c7898b960a515c7a8456bbe14b812d0bec3ee3/packages/flutter_tools/test/src/common.dart#L314-L319

seems to explain why a simple custom fake class wasn't used here.